router/server.go

```diff
@@ -102,6 +103,46 @@ func main() {
 	apiPort := flag.String("api-port", "", "api listen port")
 	flag.Parse()
 
+	httpPorts := []int{*httpPort}
+	httpsPorts := []int{*httpsPort}
+	if portRaw := os.Getenv("DEFAULT_HTTP_PORT"); portRaw != "" {
+
+		if port, err := strconv.Atoi(portRaw); err != nil {
+			shutdown.Fatalf("Invalid DEFAULT_HTTP_PORTS: %s", err)
+		} else if port == 0 {
+			log.Warn("Disabling HTTP acccess (DEFAULT_HTTPS_PORT=0)")
```

Comment:
I'm not sure I understand what you mean. The log message says "disabling HTTP access" but then `httpPorts` still contains a port, so what exactly have we disabled?